### PR TITLE
Python 3.4/5/6 and PyPy support.

### PR DESCRIPTION
**Pull request by jamadden**
- Modern .travis.yml that enables coveralls.
- 100% coverage.
- Add tox.ini.
- Rename .txt -> .rst for rendering
- Drop test dependencies on zope.app.zcmlfiles and zope.app.testing.

### CODE REVIEW

File: src/zc/resourcelibrary/tests/tests.py [hunk 1]
```diff
@@ -100,41 +104,67 @@ def zpt(s, view=None, content_type=None):
...
+        'duplicate_declarations.rst',
+        'localsite.rst',
+        globs={
+            'zcml': zcml,
+            'zpt': zpt,
+            'getRootFolder': ResourceLibraryFunctionalLayer.getRootFolder,
+        },
+        optionflags=(doctest.NORMALIZE_WHITESPACE
+                     | doctest.ELLIPSIS
+                     | renormalizing.IGNORE_EXCEPTION_MODULE_IN_PYTHON2 ),
```
Comment (mgedmin) [inline src/zc/resourcelibrary/tests/tests.py]: The unnecessary space in front of `)` jumps out at me.

**Review by mgedmin — APPROVED** [review]
~~ tl;dr~~ lgtm

**Comment by mgedmin** [comment]
You have PyPI rights now.

**Comment by jamadden** [comment]
Thank you.

**Pull request merged**